hlink/linking/model_exploration/link_step_train_test_models.py

```diff
@@ -52,161 +472,123 @@ def _run(self) -> None:
         dep_var = config[training_conf]["dependent_var"]
         id_a = config["id_column"] + "_a"
         id_b = config["id_column"] + "_b"
-        desc_df = _create_desc_df()
+
         columns_to_keep = [id_a, id_b, "features_vector", dep_var]
         prepped_data = (
             self.task.spark.table(f"{table_prefix}training_vectorized")
             .select(columns_to_keep)
             .cache()
         )
 
+        # Stores suspicious data
         otd_data = self._create_otd_data(id_a, id_b)
 
-        n_training_iterations = config[training_conf].get("n_training_iterations", 10)
+        outer_fold_count = config[training_conf].get("n_training_iterations", 10)
+        inner_fold_count = 3
```

Comment:
We may want to rename `n_training_iterations` to `num_outer_folds` and add a `num_inner_folds` attribute to the config.

Reply:
Yes,  definitely. Let's make that another  PR soon.